tests/expressions_case/tests.py

```diff
@@ -1148,6 +1148,47 @@ def test_m2m_reuse(self):
             lambda x: x[1:]
         )
 
+    def test_group_by_on_empty_case(self):
+        # Test without default value
+        self.assertQuerysetEqual(
+            CaseTestModel.objects.values('string')
+            .annotate(test=Case(output_field=IntegerField()), integer_sum=Sum('integer'))
+            .order_by('string'),
```

Comment:
Use hanging indentation.

Reply:
I'm in the habit of using black to format code (except string literals), in this case it got a bit weird indeed. 👍